File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/NameElements.java
package com.babestudios.companyinfouk.data.model.persons;


import com.google.gson.annotations.SerializedName;

public class NameElements {
	@SerializedName("surname")
	public String surname;
	@SerializedName("title")
	public String title;
	@SerializedName("middle_name")
	public String middleName;
	@SerializedName("forename")
	public String forename;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/NameElements.kt
package com.babestudios.companyinfouk.data.model.persons


import com.google.gson.annotations.SerializedName

class NameElements {
	@SerializedName("surname")
	var surname: String? = null
	@SerializedName("title")
	var title: String? = null
	@SerializedName("middle_name")
	var middleName: String? = null
	@SerializedName("forename")
	var forename: String? = null
}
